src/components/Users/UserForm.tsx

```diff
@@ -442,54 +442,57 @@ export default function UserForm({
                       />
                     </div>
                   </FormControl>
-                  {isUsernameFieldFocused ? (
-                    <>
-                      <div
-                        className="text-small mt-2 pl-2 text-secondary-500"
-                        aria-live="polite"
-                      >
-                        {(isUsernameChecking || !isUsernameTaken) && (
-                          <ValidationHelper
-                            isInputEmpty={!field.value}
-                            successMessage={t("username_success_message")}
-                            validations={[
-                              {
-                                description: "username_min_length_validation",
-                                fulfilled: (field.value || "").length >= 4,
-                              },
-                              {
-                                description: "username_max_length_validation",
-                                fulfilled: (field.value || "").length <= 16,
-                              },
-                              {
-                                description: "username_characters_validation",
-                                fulfilled: /^[a-z0-9._-]*$/.test(
-                                  field.value || "",
-                                ),
-                              },
-                              {
-                                description: "username_start_end_validation",
-                                fulfilled: /^[a-z0-9].*[a-z0-9]$/.test(
-                                  field.value || "",
-                                ),
-                              },
-                              {
-                                description: "username_consecutive_validation",
-                                fulfilled: !/(?:[._-]{2,})/.test(
-                                  field.value || "",
-                                ),
-                              },
-                            ]}
-                          />
-                        )}
-                      </div>
-                      <div className="pl-2">
-                        {renderUsernameFeedback(usernameInput || "")}
-                      </div>
-                    </>
-                  ) : (
+
+                  <>
+                    <div
+                      className={`text-small mt-2 pl-2 text-secondary-500 ${isUsernameFieldFocused ? "block" : "hidden"}`}
```

Comment:
use cn function for this and ones below

Reply:
@gobardan1707 you've only done the change in one place!

Like already said, do for other places as well wherever applicable!

Please do a self-review of the code!